test/parallel/test-string-decoder.js

```diff
@@ -55,9 +55,14 @@ assert.strictEqual(decoder.write(Buffer.from('\ufffd\ufffd\ufffd')),
 assert.strictEqual(decoder.end(), '');
 
 decoder = new StringDecoder('utf8');
-assert.strictEqual(decoder.write(Buffer.from('efbfbde2', 'hex')), '\ufffd');
+assert.strictEqual(decoder.write(Buffer.from('EFBFBDE2', 'hex')), '\ufffd');
```

Comment:
I don't understand. The hex string is being passed to the Buffer creation function (`Buffer.from()`). That function has plenty of lowercase and uppercase hex string tests in the appropriate `test-buffer*` tests.